bootloader/assets/grub_test.go

```diff
@@ -58,3 +58,22 @@ func (s *grubAssetsTestSuite) TestGrubRecoveryConf(c *C) {
 		"snapd_recovery_system",
 	)
 }
+
+func (s *grubAssetsTestSuite) TestGrubCmdlineSnippet(c *C) {
```

Comment:
I would have two tests:

* test that check the snippets for edition 1
* test about cross checking the assets vs the corresponding snippets